azure-pipelines.yml

```diff
@@ -156,14 +156,14 @@ stages:
 
           - task: Docker@2
             displayName: 'Push ACR [latest]'
+            condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
```

Comment:
Spis ze zvedavosti - obsah te condition mi od pohledu prijde stejny, ma to umisteni nejaky vliv (fakt doufam, ze ne :D)

Reply:
Ne nema, dal jsem ji jen na stejný místo jako u ostatních tasku, protože jsem ji dvakrát přehlídl